tests/dummy/app/router.js

```diff
@@ -12,7 +12,11 @@ Router.map(function() {
     this.route('usage');
     this.route('quickstart');
     this.route('patterns');
-    this.route('deploying');
+    this.route('deploying', function() {
+      // This exists so we can acceptance test whether the docs pages tolerate
+      // embedded child routes. It's not used in the public documentation site.
+      this.route('test-nested-route');
```

Comment:
Can we add this to the sandbox app instead? The sandbox app is meant to be our "scenarios" app, where we put things that are dummy content which only exist for testing and demonstration purposes

Reply:
Oh yeah, definitely, I didn't know that existed and felt vaguely bad about this already.